packages/info-sprinkle/src/InfoSprinkle/InfoSprinkle.tsx

```diff
@@ -30,13 +30,18 @@ export const InfoSprinkle = React.forwardRef<
         baseFontSize={baseFontSize}
         trigger={
           <span
```

Comment:
https://github.com/mongodb/leafygreen-ui/pull/1999#discussion_r1347643174. We used span because of an example we found but that example no longer uses a `span`, it uses a `button`.